### PR TITLE
End game

### PR DESCRIPTION
**Pull request by vicphan**
In this PR:
* displays card that shows end of game, thanks for playing message
* resets all flags and states when game ends
* add return to main menu button on last question
* remove cookies when game ends
* minor aesthetic changes

### CODE REVIEW

File: src/hooks/useClearCookies.ts [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import Cookies from 'js-cookie';
+
+export function useClearCookies() {
+    // Removes cookies
+    Cookies.remove('secondsLeft');
+    Cookies.remove('gameCode');
+    Cookies.remove('isHost');
+
```
Comment (sebcontreras) [inline src/hooks/useClearCookies.ts:8]: Thanks for adding this! There are a few more cookies now from the 'client rejoin' PR. I'll add them in once this is merged

File: src/state/questionSlice.ts [hunk 2]
```diff
@@ -39,20 +39,36 @@ const questionStateSlice = createSlice({
...
+    incrementQuestionIndex: (state: QuestionState, action: PayloadAction<number>): void => {
       if (state.index !== undefined) {
         state.index = state.index + action.payload;
       }
     },
     resetQuestionScore: (state: QuestionState, action: Action): void => {
       state.questionScore = undefined;
-    }
+    },
+    resetQuestionState: (state: QuestionState, action: Action): void => {
```
Comment (sebcontreras) [inline src/state/questionSlice.ts:53]: nice

**Review by sebcontreras — APPROVED** [review]
Looks good to me!

**Review by desileal — APPROVED** [review]
looks good! I can't test because vercel is down and I don't think this branch is merged into heroku but the logic seems right to me

**Pull request merged**